### DISCUSSION TITLE
How to use provide / inject?

### DISCUSSION THREAD

**Discussion by bartenra** [histoire-dev/histoire#135] [Q&A] [answered]
In our `App.vue` we `inject()` some things that some components need (and `provide()` in their setup). How can I inject stuff on a global level with Histoire? I don't want to wrap all my stories in another component ideally.

**Comment by hugoattal** [comment, marked as answer]
Hey! Did you take a look at the `defineSetupVue3` function? This might suit your need.
https://histoire.dev/guide/vue3/app-setup.html

**Reply by bartenra** [reply]
`app.provide` -> did not know that was a thing.
